lib/iris/tests/unit/analysis/test_PERCENTILE.py

```diff
@@ -307,14 +314,14 @@ def test_masked(self):
             as_concrete_data(actual)
 
     def test_masked_mdtol_0(self):
-        shape = (2, 11)
+        shape = (3, 11)
```

Comment:
Mismatched comment above may have been intended for here.

Reply:
https://github.com/SciTools/iris/pull/4755/commits/7af47a14fce0bad6269c234ece83c43b5f0d359f